experimental/WreathMacdonaldpols/docs/src/introduction.md

```diff
@@ -0,0 +1,43 @@
+```@meta
+CurrentModule = Oscar
+CollapsedDocStrings = true
+DocTestSetup = Oscar.doctestsetup()
+```
+
+# Wreath Macdonald polynomials
+
+The existence, integrality and positivity of wreath Macdonald polynomials
+ has been conjectured by Haiman [Hai02](@cite) and proved by Bezrukavnikov
+ and Finkelberg [BF14](@cite). Here we have implemented an algorithm which
+ computes the wreath Macdonald polynomials as defined in the review by
+ Orr and Shimozono on this topic [OS23](@cite).
+
+Wreath Macdonald polynomials depend on two parameters. The first one is
+ a multipartition since they are indexing the set of irreducible representations of
+ the complex reflection group ``G(r,1,n)``. The second one is an element
+ of the affine Weyl group of type ``A_r`` which is isomorphic to the semi-direct
+ product of the finite Weyl group of type ``A_r`` (the symmetric group on ``r`` letters)
+ and of the coroot lattice of type ``A_r``. The element of the coroot lattice is
+ given in the canonical basis. It is then the sublattice of ``\mathbb{Z}^r`` of elements summing
+ up to zero.
+
+```@docs
+wreath_macs(n::Int, r::Int, wperm::PermGroupElem, coroot::Vector{Int})
+wreath_mac(lbb::Multipartition, wperm::PermGroupElem, coroot::Vector{Int})
```

Comment:
This is not meant to block this PR (the code is in experimental and so we can still change it post-merge!) but just to point this out: normally we try to minimize abbreviations like this and would just call these `wreath_macdonald_polynomials` and `wreath_macdonald_polynomial` with the understanding that everyone is either using the REPL with tab completion, or an editor with some completion mechanism, so the length is not a major issue.

If an abbreviation really is to be used, it better reflect what kind of object is returned. So like `wreath_mac_poly` or `wr_mac_poly`.


Thoughts @fieker @ulthiel ?

Reply:
I am against abbreviations, in general. :-)